ts/nni_manager/training_service/reusable/environments/amlEnvironmentService.ts

```diff
@@ -126,22 +101,24 @@ export class AMLEnvironmentService extends EnvironmentService {
             amlEnvironment.command = `mv envs outputs/envs && cd outputs && ${amlEnvironment.command}`;
         }
         amlEnvironment.command = `import os\nos.system('${amlEnvironment.command}')`;
-        amlEnvironment.useActiveGpu = this.amlClusterConfig.useActiveGpu;
-        amlEnvironment.maxTrialNumberPerGpu = this.amlClusterConfig.maxTrialNumPerGpu;
+        amlEnvironment.useActiveGpu = false;
```

Comment:
does not support setting useActiveGpu?

Reply:
Do we really need it? I thought this option does not make sense for AML.